### ISSUE TITLE
Why is getattr(get_vec_normalize(envs), 'ob_rms', None) included in save_model?

### ISSUE THREAD

**Issue by Akella17** [ikostrikov/pytorch-a2c-ppo-acktr#167]
I would like to know why is getattr(get_vec_normalize(envs), 'ob_rms', None) saved along with actor-critc network?
*line: 154-155, main.py*

**Comment by ikostrikov** [comment]
Because input normalization is computer over all observed trajectories and it would not be possible to recompute it after loading the model.

**Comment by Akella17** [comment]
Oh got it! Did not observe that detail. Thanks for the quick response!!

**Comment by Akella17** [comment]
Is there any reason why we are saving the entire model, rather than its state_dict()?
